### PR TITLE
Support creating api document of maflib/ from docstring

### PR DESCRIPTION
**Pull request by hiroshinoji**
sphinx で api document を出力するための conf.py の修正です。

### CODE REVIEW

File: document/source/conf.py [hunk 1]
```diff
@@ -16,7 +16,35 @@
...
+    if os.path.exists(waf_path):
+        import subprocess
+        devnull = open('/dev/null')
+        # execute waf to create waflib directory
+        subprocess.call(['python', waf_path], stdout=devnull, stderr=devnull)
+        waflib = waflib_cands(waf_root)[0]
+    else:
+        raise Exception("waf script is missing. Please place it to the project root directory.")
+
+if waflib:
```
Comment (beam2d) [inline document/source/conf.py]: You can omit this `if`.

**Comment by beam2d** [comment]
See also: #6

File: document/source/conf.py [hunk 2]
```diff
@@ -16,7 +16,35 @@
...
+    sys.path.insert(0, waflib)
+    sys.path.insert(0, os.path.join(waflib, "waflib"))
```
Comment (beam2d) [inline document/source/conf.py]: Is the second insertion required?

File: document/source/conf.py [hunk 3]
```diff
@@ -16,7 +16,35 @@
...
+
+maflib = os.path.join(waf_root, "maflib")
+sys.path.insert(0, waf_root)
+sys.path.insert(0, maflib)
```
Comment (beam2d) [inline document/source/conf.py]: ditto.

File: document/source/conf.py (hunk 1 above)
Comment (hiroshinoji) [inline document/source/conf.py]: done.

File: document/source/conf.py (hunk 2 above)
Comment (hiroshinoji) [inline document/source/conf.py]: Deleted.

**Comment by beam2d** [comment]
LGTM

**Pull request merged**